builtin/logical/pki/path_root.go

```diff
@@ -109,9 +109,10 @@ func pathSignSelfIssued(b *backend) *framework.Path {
 				Type:        framework.TypeString,
 				Description: `PEM-format self-issued certificate to be signed.`,
 			},
-			"allow_different_signature_algorithm": &framework.FieldSchema{
+			"require_matching_certificate_algorithms": &framework.FieldSchema{
 				Type:        framework.TypeBool,
-				Description: `If true, allow the public key type of the signer to differ from the self issued certificate.`,
+				Default:     false,
+				Description: `If true, allow the public key algorithm of the signer to differ from the self issued certificate.`,
```

Comment:
I don't think this description is correct? Note that we switched the order of this variable, so I think the description would be

> if true, require the public key algorithm ... to match ...

Reply:
Yep, whoops.

Reply:
Weird, my local codebase doesn't read like that.